rasa_nlu/data_router.py

```diff
@@ -180,8 +206,7 @@ def parse(self, data):
         model = self.model_store[alias]
         response = model.parse(data['text'], data.get('time', None))
         if self.responses:
-            log = {"user_input": response, "model": alias, "time": datetime.datetime.now().isoformat()}
-            self.responses.info(json.dumps(log, sort_keys=True))
+            self.responses.info(user_input=response, model=alias)
```

Comment:
I removed it because the `timestamp` is already part of Twisted Logger object output. No duplicate there.